docs/pages/2019_MARVEL_Psik_MaX/sections/verdi_cmdline.rst

```diff
@@ -39,406 +54,385 @@ The first output line should look like
+In order to print a list with all processes that have already finished, you can use the ``-S/--process-state`` flag as follows:
+
+.. code:: console
+
+    verdi process list -S finished
+
+This command will list all the processes that have a process state ``Finished`` and should look something like:
+
+.. code:: console
+
+    PK    Created    State           Process label    Process status
+    ----  ---------  --------------  ---------------  ----------------
+    1178  1653D ago  ⏹ Finished [0]  PwCalculaton
+    1953  1653D ago  ⏹ Finished [0]  PwCalculaton
+    1734  1653D ago  ⏹ Finished [0]  PwCalculaton
+     336  1653D ago  ⏹ Finished [0]  PwCalculaton
+    1056  1653D ago  ⏹ Finished [0]  PwCalculaton
+    1369  1653D ago  ⏹ Finished [0]  PwCalculaton
+
+    Total results: 6
+
+    Info: last time an entry changed state: never
+
+The 'Finished' state is not the only state that a process can have, but a process can have any of the following states:
+
+    * ``Created``
+    * ``Waiting``
+    * ``Running``
+    * ``Finished``
+    * ``Excepted``
+    * ``Killed``
+
+The first three states are 'active' states, meaning the process is not done yet, and the last three are 'terminal' states.
+Once a process is in a terminal state, it will never become active again.
+The `official documentation <https://aiida-core.readthedocs.io/en/latest/concepts/processes.html#process-state>`_ contains more details on process states.
+
+If, instead of any one particular process state, you want to list processes in *any* state, you can use the ``-a/--all`` flag instead:
+
+.. code:: console
+
+    verdi process list -a
+
+This command will list all the processes that have *ever* been launched.
+As your database will grow, the output will become so large that it becomes unusable.
+To limit the number of results, you can use the ``-p/--past-days <NUM>`` option, that will only show processes that were created ``NUM`` days ago.
+For example, if you want to list all the process launched since the last day, you can execute:
 
 .. code:: console
 
-    verdi process list --process-state finished
+    verdi process list -a -p1
 
-Another very typical option combination allows to get processes in
-*any* state (flag ``-a/--all``) generated in the past ``NUM`` days
-(``-p/--past-days <NUM>``): e.g., for process in the past 1 day:
-``verdi process list -p1 -a``. Each row of the output identifies a process and
-shows some information about it. For a more detailed list of properties, choose
-one row by noting down its PK (primary key) number (first column of the output)
-and type in the terminal
+Each row of the output identifies a process with some basic information about its status.
+For a more detailed list of properties, you can use ``verdi process show``, but to address any specific process, you need an identifier for it.
+An identifier for any entity in AiiDA comes in three different forms:
+
+ * PK
```

Comment:
What about concentrating this a bit as follows:
```suggestion
 * "Primary Key" (PK): An integer, e.g. ``723``, that identifies your entity within your database (automatically assigned)
 * `Universally Unique Identifier <https://en.wikipedia.org/wiki/Universally_unique_identifier#Versions_3_and_5_(namespace_name-based)>`_ (UUID): A string, e.g. ``ce81c420-7751-48f6-af8e-eb7c6a30cec3`` that identifies your entity globally  (automatically assigned)
  * Label: A string, e.g. ``test_process`` that allows you to assign a human readable label
```
Also added a link to the definition of uuid (are we using type 3?)

Reply:
Most of the text below is still relevant, but it can be shortened a bit

Reply:
We are using type 4

Reply:
Done and adapted the link